### ISSUE TITLE
bone marrow maturation arrest

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5576]
**Preferred term label:**
bone marrow maturation arrest at the promyelocytic/myelocytic stage

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/19036076/

**Parent term (use hpo.jax.org/app)**

https://hpo.jax.org/app/browse/term/HP:0012135

but this will need some rearrangement.  This term is bundled
https://hpo.jax.org/app/browse/term/HP:0410253 and should be split

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@nicolevasilevsky This is a bundled term
https://hpo.jax.org/app/browse/term/HP:0410253
I am going to rename it 
myeloid maturation arrest in bone marrow

If there is a need to re-annotate data, then one could add "Neutropenia"
